DigitalOffice/Services/FileService/Commands/GetFileByIdCommand.cs

```diff
@@ -0,0 +1,29 @@
+﻿using LT.DigitalOffice.FileService.Commands.Interfaces;
+using LT.DigitalOffice.FileService.Database.Entities;
+using LT.DigitalOffice.FileService.Mappers.Interfaces;
+using LT.DigitalOffice.FileService.Models;
+using LT.DigitalOffice.FileService.Repositories.Interfaces;
+using Microsoft.AspNetCore.Mvc;
+using System;
```

Comment:
First connect system namespaces and than namespaces created in the our project

Reply:
Fixed